### DISCUSSION TITLE
External monitor completely dark, cannot increase brightness

### DISCUSSION THREAD

**Discussion by aaron-resnick** [MonitorControl/MonitorControl#1411] [Q&A]
Hello, I recently installed MonitorControl on my M1 MacBook Pro (Apple Silicon) and I was using it to control and Dell U2520D external monitor. I have MacOS Ventura 13.4.1 installed, and I was using the latest version of Monitor Control.

At first everything was working great. But only a few hours after installing it, I got up to go to the bathroom, and when I came back, my monitor -- which had been working perfectly before -- was all of a sudden completely dark. I tried to use MonitorControl to bring the brightness back up, but nothing that I did was able to get the brightness to return.

I can tell that the monitor is still functioning, since for example when I unplug my computer from the monitor and then plug it back in, I can see the windows on my computer "disappear" when the computer is shifting those windows to the external monitor, and then the windows come back and appear on my laptop screen when I unplug the computer from the monitor. 

So the monitor is working, but it is somehow just stuck at 0 brightness. I have tried unplugging the monitor and plugging it back in, uninstalling monitor control, etc. but I have not had any luck. 

Does anyone know what might be happening? My monitor is completely useless at the moment, so any help would be appreciated!

**Comment by kiren-rawatlal** [comment]
I'm having the same issue.
After a while of use, (not inactivity), it starts to go dimmer, progressively with with second - i can see it each second getting darker over about 15seconds. 
I installed it on 2 computers - both apple silicon - M2Pro and M1air - same issue on both.
To resolve, either reconnect the cable or go to the primary monitor and tap the brightness setting on the external monitor - and it resets to the latest brightness.

This is an awesome app - but what is this bug.
